.github/workflows/ruby-ci.yml

```
@@ -3,6 +3,11 @@ name: Ruby CI
 on:
   workflow_call:
     inputs:
+      run:
```

No need to change, but `run` felt a bit generic

```suggestion
      run-command:
```

---

I was about to name it `run-command` first, but I was aiming for something short, and `run` is also the same keyword Actions is using, and it allows for multiple commands with the 

```
run: |
  ...
  ...
```

syntax

So my vote goes to `run`